bbb-common-web/src/main/scala/org/bigbluebutton/api2/MsgBuilder.scala

```diff
@@ -83,7 +84,12 @@ object MsgBuilder {
 
     val urls = Map("thumb" -> thumbUrl, "text" -> txtUrl, "svg" -> svgUrl, "png" -> pngUrl)
 
-    try {
+    val result = Using.Manager { use =>

```

Comment:
Was this part added wrongly?

Reply:
This was added to address the issue of the stream and reader not being closed.